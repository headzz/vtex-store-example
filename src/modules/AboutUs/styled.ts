import styled from 'styled-components'

export const AboutUsContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;

  @media (min-width: 900px) {
    flex-direction: row;
    justify-content: space-evenly;
  }
`
