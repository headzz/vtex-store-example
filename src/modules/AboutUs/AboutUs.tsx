import Layout from 'components/Layout'

import LeftArea from './components/LeftArea'
import RightArea from './components/RightArea'
import { AboutUsContainer } from './styled'

/**
 * @component
 * @name AboutUs
 * @description creates AboutUs component section to use on screen.
 * @namespace components modules/AboutUs
 *
 */
const AboutUs: React.FC = () => (
  <Layout.WhoWeAre>
    <AboutUsContainer>
      <LeftArea />
      <RightArea />
    </AboutUsContainer>
  </Layout.WhoWeAre>
)

export default AboutUs
