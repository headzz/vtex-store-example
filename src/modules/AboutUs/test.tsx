import { render } from '@testing-library/react'

import AboutUs from './AboutUs'

describe('AboutUs', () => {
  it('should render AboutUs', () => {
    const { container } = render(<AboutUs />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
