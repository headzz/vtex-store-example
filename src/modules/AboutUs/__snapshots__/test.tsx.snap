// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs should render AboutUs 1`] = `
.c0 {
  background: 0;
  background-color: #ffffff;
  padding: 6.4rem 0;
  width: 100%;
}

.c5 {
  background-color: transparent;
  border-radius: 6px;
  border: 1px solid #000000;
  color: #000000;
  font-size: 1.6rem;
  margin-left: 0;
  outline: 0;
  padding: 1.2rem 2.4rem;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 {
  color: #000000;
  font-size: 3.2rem;
  font-weight: 700;
}

.c4 {
  color: #000000;
  font-size: 1.6rem;
  font-weight: 500;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: 1.6rem;
  position: relative;
  padding: 0 4rem;
  width: 100%;
}

.c2 p {
  margin-bottom: 2.4rem;
}

.c2 a {
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
}

.c6 {
  margin: 0 auto;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

@media (min-width:1024px) {
  .c0 {
    background: linear-gradient( 90deg,#ffffff 50%,#F2F2F2 50% );
  }
}

@media (min-width:1024px) {
  .c2 {
    -webkit-align-items: flex-start;
    -webkit-box-align: flex-start;
    -ms-flex-align: flex-start;
    align-items: flex-start;
    -webkit-box-pack: space-evenly;
    -webkit-justify-content: space-evenly;
    -ms-flex-pack: space-evenly;
    justify-content: space-evenly;
    padding: 0;
    width: 24.2rem;
  }
}

@media (min-width:1024px) {
  .c6 {
    margin: 0;
  }
}

@media (min-width:900px) {
  .c1 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-pack: space-evenly;
    -webkit-justify-content: space-evenly;
    -ms-flex-pack: space-evenly;
    justify-content: space-evenly;
  }
}

<section
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    >
      <p
        class="c3"
        color="#000000"
        title="small"
      >
        OLÁ, SOMOS A JÜSSI
      </p>
      <p
        class="c4"
        color="#000000"
        title=""
      >
        A Jüssi é uma agência integrante do grupo global WPP que vem há 10 anos consolidando o pensamento voltado para produtos e resolução de problemas. Nosso área dedicada exclusivamente para Produtos Digitais é organizada em 6 especialidades: Product Managamenet, User Experience Design, SEO, Tecnologia, Agile e User Behavior Analytics.
      </p>
      <a
        class="c5"
        href="https://jussi.com.br/about.html"
      >
        Conheça a Jüssi
      </a>
    </div>
    <div
      class="c6"
    >
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            role="presentation"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjc2IiBoZWlnaHQ9IjUyMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <img
          alt="banner"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="visibility: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
      </div>
    </div>
  </div>
</section>
`
